SharedLibs/TileMapHelper/src/render/TileMapManager.ts

```diff
@@ -117,6 +117,27 @@ export class TileMapManager {
     );
   }
 
+  getOrLoadSimpleTileMap(
+    tileMapAsJsObject: object,
+    objectName: string,
+    tileSize: number,
+    tileSetColumnCount: number,
+    tileSetRowCount: number,
+    // Logic using callback has been set up to mimic what has been done for other
+    // loading methods. But it could be refactored to directly return the tile map.
+    callback: (tileMap: EditableTileMap) => void
+  ): void {
+    // TODO: Is it useful to cache the tilemap since it belongs to an instance?
```

Comment:
I'm not sure, but i'd say it wouldn't hurt?

Reply:
The thing is that we only have the ptr to differentiate instances. And the ptr is unique. And this method is called only once at initialization in the runtime. So it will always be useless. Unless I'm missing something